pkg/scheduler/util/utils.go

```diff
@@ -145,4 +145,4 @@ func ClearNominatedNodeName(cs kubernetes.Interface, pods ...*v1.Pod) utilerrors
 func IsScalarResourceName(name v1.ResourceName) bool {
 	return v1helper.IsExtendedResourceName(name) || v1helper.IsHugePageResourceName(name) ||
 		v1helper.IsPrefixedNativeResource(name) || v1helper.IsAttachableVolumeResourceName(name)
-}
+}
```

Comment:
Please revert this change. (leave the empty line there)

Reply:
fine